guides/typed-json/json-data-bindings/README.md

```diff
@@ -177,21 +177,35 @@ For example:
 <div class="code-name">PersonPage.json.cs</div>
 
 ```cs
-using Starcounter;
+partial class PersonPage : Json
+{
+    static PersonPage()
+    {
+        DefaultTemplate.FullName.Bind = null;
+    }
+}
+```
 
-namespace MyApp
+By applying this to the example in the [bindings to database objects section](#binding-to-database-objects), the resulting JSON would be `{"FirstName":"Steve","LastName":"Smith","FullName":""}`. Since the `FullName` property is not bound, it will not contain any value.
+
+#### Binding to Properties With Different Names
+
+If a property should be bound to a property that has a different name than the property in the JSON, a binding value can be set.
+
+For example, to bind `FirstName` to `LastName` in the example in the [bindings to database objects section](#binding-to-database-objects) and vice versa, the following code can be used:
+
+```cs
+public class PersonPage : Json
 {
-    partial class PersonPage : Json
+    static PersonPage()
     {
-        static PersonPage()
-        {
-            DefaultTemplate.FullName.Bind = null;
-        }
+        DefaultTemplate.FirstName.Bind = "LastName";
+        DefaultTemplate.LastName.Bind = "FirstName";
```

Comment:
Might want to specify what represents the DefaultTemplate.FirstName.Bind and what represents the property name as a string value. Does "LastName" come from the object set in the Data property?